hcl2template/types.packer_config.go

```diff
@@ -84,7 +87,7 @@ func (cfg *PackerConfig) EvalContext(variables map[string]cty.Value) *hcl.EvalCo
 			}),
 			buildAccessor: cty.UnknownVal(cty.EmptyObject),
 			packerAccessor: cty.ObjectVal(map[string]cty.Value{
-				"version": cty.StringVal(version.FormattedVersion()),
```

Comment:
I think that this hcl2template will continue living in the core repo so we don't need to do that here